docs/blocking-schema.rst

```diff
@@ -0,0 +1,219 @@
+* `An LSH-Based Blocking Approach with a Homomorphic Matching Technique for Privacy-Preserving Record Linkage <https://www.computer.org/csdl/journal/tk/2015/04/06880802/13rRUxASubY>`_
+
+The format of the linkage schema is defined in a separate
+`JSON Schema <https://json-schema.org/specification.html>`_ specification document -
+`blocking-schema.json <https://github.com/data61/anonlink-client/blob/master/docs/schemas/blocking-schema.json>`_.
+
+Basic Structure
+---------------
+
+A linkage schema consists of three parts:
+
+* :ref:`type <blocking-schema/type>`, the blocking method to be used
+* :ref:`version <blocking-schema/version>`, the version number of the hashing schema.
+* :ref:`config <blocking-schema/config>`, an json configuration of that blocking method that varies with different blocking methods
+
+
+Example Schema
+--------------
+
+::
+
+    {
+      "type": "lambda-fold",
+      "version": 1,
+      "config": {
+        "blocking-features": [1, 2],
+        "Lambda": 30,
+        "bf-len": 2048,
+        "num-hash-funcs": 5,
+        "K": 20,
+        "input-clks": true,
+        "random_state": 0
+      }
+    }
+
+Schema Components
+-----------------
+.. _blocking-schema/type:
+
+type
+~~~~
+String value which describes the blocking method.
+
+================= ================================
+name              detailed description
+================= ================================
+p-sig             Probability Signature blocking method from `Scalable Entity Resolution Using Probabilistic Signatures on Parallel Databases <https://arxiv.org/abs/1712.09691>`_
+lambda-fold       LSH based Lambda Fold Redundant blocking method from `Scalable Entity Resolution Using Probabilistic Signatures on Parallel Databases <https://arxiv.org/abs/1712.09691>`_
+================= ================================
+
+.. _blocking-schema/version:
+
+version
+~~~~~~~
+
+Integer value that indicates the version of blocking schema
+
+.. _blocking-schema/config:
+
+config
+~~~~~~
+
+A dictionary of configuration to use different blocking methods
+Next we will detail the specific configuration for supported blocking methods.
+
+Specific configuration of supported blocking methods can be found here:
+
+- `config of p-sig <blocking-schema/p-sig>`
+- `config of lambda-fold <blocking-schema/lambda-fold>`
+
+.. _blocking-schema/p-sig:
+
+config of p-sig
+~~~~~~~~~~~~~~~
+===================== ============= ==========================
+configuration         type          description
+===================== ============= ==========================
+blocking-features     list[integer] specify which features u
+filter                dictionary    filtering threshold
+blocking-filter       dictionary    type of filter to generate blocks
+signatureSpecs        list of lists signature strategies where each list is a combination of signature strategies
```

Comment:
suggest surrounding the attribute name in backticks to make it clear they are the machine readable values/names

Reply:
I would prefer just plain text since it makes table looks neat. User will understand it is machine readable given the full config example. I checked the docs for clkhash linkage schema, attributes are also in plain text

Reply:
that sounds reasonable 👍